exercises/practice/kindergarten-garden/.meta/Example.roc

```diff
@@ -0,0 +1,32 @@
+module [plants]
+
+Student : [Alice, Bob, Charlie, David, Eve, Fred, Ginny, Harriet, Ileana, Joseph, Kincaid, Larry]
+Plant : [Grass, Clover, Radishes, Violets]
+
+students : List Student
+students = [Alice, Bob, Charlie, David, Eve, Fred, Ginny, Harriet, Ileana, Joseph, Kincaid, Larry]
+
+studentIndex : Student -> U64
+studentIndex = \student ->
+    when students |> List.findFirstIndex \s -> s == student is
+        Ok index -> index
+        Err NotFound -> crash "Unreachable: all possible students are present in the students list"
```

Comment:
I suggest pattern matching on student here and returning the index directly instead. That way the impossible case won't be necessary and the students array can be removed also.

Reply:
Excellent suggestion, thanks Isaac!